doc/en/pools.rst

```diff
@@ -331,45 +331,72 @@ in modules that matches the name_pattern under current working directory.
 
 .. code-block:: python
 
-    @task_target()
-    def make_multitest1():
-        # a test target shall only return 1 runnable object
```

Comment:
This line literally means "a test target SHALL only return 1 runnable object", as it only return 1 Multitest object.

Reply:
put it back.